src/controllers/queries/index.ts

```diff
@@ -246,32 +312,22 @@ export class QueriesController extends HTMLController {
       companyId: UUID
       action: 'success'
       totalScope3CarbonEmissions: string
-      partialResponse?: number
     }
   ): Promise<HTML> {
     this.logger.debug('query page requested')
 
-    const [connection]: ConnectionRow[] = await this.db.get(
-      'connection',
-      { id: body.companyId, status: 'verified_both' },
-      [['updated_at', 'desc']]
-    )
+    const [connection] = await this.db.get('connection', { id: body.companyId, status: 'verified_both' }, [
+      ['updated_at', 'desc'],
+    ])
     if (!connection) {
-      throw new InvalidInputError(`Invalid connection ${body.companyId}`)
+      throw new NotFoundError(`Invalid connection ${body.companyId}`)
```

Comment:
This should be an `InvalidInputError`. `NotFoundError` is reserved for path parameter resources only

Reply:
my bad, updated.